.github/workflows/next-build.yaml

```diff
@@ -216,3 +216,12 @@ jobs:
           echo "Using version ${{ needs.tag.outputs.desktopVersion }}"
           sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${{ needs.tag.outputs.desktopVersion }}\",#g" tests/playwright/package.json
           cd tests/playwright && pnpm build && pnpm publish --tag next --no-git-checks --access public
+
+      - name: Publish Podman Extension API to npmjs
+        run: |
+          echo "Using version ${{ needs.tag.outputs.desktopVersion }}"
+          sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${{ needs.tag.outputs.desktopVersion }}\",#g" extensions/podman/packages/api/package.json
+          cd extensions/podman/packages/api
+          echo "Install latest @podman-desktop/api@${{ needs.tag.outputs.desktopVersion }}"
```

Comment:
Seems to me the package is not the good one ?

Reply:
this is attempt to publish podman extension api package with dependency to the same podman-desktop/api version that is being built